### ISSUE TITLE
Feature request: Add music to credits

### ISSUE THREAD

**Issue by atombraiderfan** [rr-/Tomb1Main#356]
As things stand, the endgame credits roll with complete silence. I think it would be nice to have music play while credits roll, for both TR1 and TRUB.

**Comment by rr-** [comment]
Which track?

**Comment by atombraiderfan** [comment]
I was thinking maybe "track02" as that's the music which plays on the main menu.  Also since the credits are quite short, is it possible to keep the track playing uninterrupted as the game transitions from the credits to the menu?

**Comment by Richard-L** [comment]
@rr- I'd like to propose some ideas for a bigger effort for a "Game completion experience", to include more of the credit images I'd posted under #247, a final game stats screen, and some music to go along (as is the intention of this issue). Would you rather a new superseding issue was made for this, or should it go in here?

**Comment by atombraiderfan** [comment]
Just to add, I noted that the Install.pcx image is only seen when installing the DOS version of TR1 (see #247 ) and thus is rarely seen nowadays, so maybe it would be fitting to re-use the image as a a backdrop for the final stats screen?  Also TR2 and TR3 both have a background image for the final stats so it's consistent.

**Comment by rr-** [comment]
I'd rather have different issues for each feature request so we can mark them off as closed and work independently on them.
Final stats screen is completely different for example to adding music, and blocked by other things.

**Comment by Richard-L** [comment]
Okay then.

It's sad there was never a final end theme track by Nathan McCree, like TR2 and 3 have them.

TR2's end theme is 2:45min, TR3's is 1:20min, so although it depends how many credit screens you want to show at the end (plus a potential final stats screen), I would say perhaps we land somewhere in the middle, and for TR1M go with "Main Theme 2" (`track04.flac` in TombATI's files), at roughly 2min?

The main theme would make certain sense as a common ground. Then again you will hear it again directly after, when the game returns to the main menu and enables NG+.

The last piece of music to play during the Natla fight is "The Earth Begins to Shake" (`track06.flac`). Since TR2 and TR3 have quite activated tracks, and since the credit screens for TR1 are showing her being her feisty self, it would seem to make sense to choose something to go along with it. So perhaps this track could just repeat over the credits?

**Comment by walkawayy** [comment]
Some options I was thinking of...
1: Tomb Raider Theme [Alternate Mix 2]
https://www.youtube.com/watch?v=GbQeZMoARDw&list=PLEeafQzOeklQJFsZk5Lk1ttHDtPxeLVGH&index=14

2: Longing For Home [Alternate Mix
https://www.youtube.com/watch?v=loQ1m_EiA3M

3: Tomb Raider Theme
The main theme seems kinda boring since it plays on the title screen which comes up right after.

**Comment by walkawayy** [comment]
Here's my credits music proposal. I think the TR Theme Alt Mix 2 works well. It's 1 of 2 unused tracks, so it's almost like a dedicated credits track. I think this feature would be good to release with the new final stats screen. And it's just a gameflow entry, so it's customizable.

https://streamable.com/e1hbzi

**Comment by rr-** [comment]
I approve of this, sounds awesome and the buildup discourages from skipping the credits.

**Comment by Richard-L** [comment]
I also love it. I have some follow up ideas on how to further capitalise on this game ending experience, but we can use separate issues/discussions for that. Super nice work @walkawayy.

But I need you to prove to me please how some tracks are unused :stuck_out_tongue:
